lib/geocoder/configuration_hash.rb

```diff
@@ -1,11 +1,11 @@
-require 'hash_recursive_merge'
-
 module Geocoder
   class ConfigurationHash < Hash
-    include HashRecursiveMerge
-
     def method_missing(meth, *args, &block)
       has_key?(meth) ? self[meth] : super
     end
+
+    def respond_to_missing?(meth, include_private = false)
```

Comment:
there's some unrelated work here to placate my editor's linter – lmk if you'd prefer to leave them out.